app/mailers/project_status_change_mailer.rb

```diff
@@ -0,0 +1,22 @@
+# Sends emails to notify project team members when their project status has changed
+# @see http://localhost:3000/rails/mailers/project_status_change_mailer/
+class ProjectStatusChangeMailer < ApplicationMailer
```

Comment:
Not in love with having the same mailer for both kinds of status changes. I'd prefer separating this back out; I'm concerned having the abstraction present in the mailers at first is going to obscure stuff (particularly around appropriate messaging structure.)

Nonblocking.

Reply:
I'd be fine pulling it back out into separate mailers. I prefer one to one mappings of mailers to actual emails sent as well; and hiding the abstraction at the locales level feels really weird to me.

I'd rather have `ProjectApprovedMailer` and `ProjectRejectedMailer` and switch between them using a boolean.